.github/workflows/tests.yml

```diff
@@ -28,11 +28,10 @@ jobs:
           - '8.1'
           - '8.2'
         redis:
-          - '3'
-          - '4'
-          - '5'
-          - '6'
-          - '7'
```

Comment:
Let's please add 7 back to the regular tests.

Reply:
Fixed